docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -101,75 +90,70 @@ FIELDS TERMINATED BY '|'
 LOCATION 'alluxio://master_hostname:port/ml-100k';
 ```
 
-## Use Alluxio as the FileSystem for External tables currently in HDFS
+The difference is that Hive will manage the lifecycle of internal tables.
+When you drop an internal table, Hive drops the table metadata and data both.
```

Comment:
"drops the table metadata and data both" -> "deletes both the table metadata and the data."